ivy_tests/test_ivy/helpers.py

```diff
@@ -1756,15 +1757,33 @@ def test_frontend_function(
         args_ivy, kwargs_ivy = ivy.args_to_ivy(*args, **kwargs)
 
     # frontend function
-    frontend_fn = ivy.functional.frontends.__dict__[frontend].__dict__[fn_tree]
+    frontend_fn = ivy.functional.frontends.__dict__[frontend].__dict__[function_name]
 
     # run from the Ivy API directly
-    if test_unsupported:
-        test_unsupported_function(fn=frontend_fn, args=args, kwargs=kwargs)
-        return
+    if instance_method:
+        if args == []:
+            instance = list(kwargs.values())[0]
+            kwargs.pop(list(kwargs.keys())[0])
+        else:
+            instance = args[0]
+            args = args[1:]
+
+        if test_unsupported:
+            test_unsupported_function(
+                fn=instance.__getattribute__(function_name), args=args, kwargs=kwargs
+            )
+
+        setattr(instance, "ivy_instance_method", frontend_fn)
```

Comment:
Issue here, can't figure out how to call the instance method for the instance array.